.github/workflows/docs-test.yml

```diff
@@ -33,6 +33,9 @@ jobs:
         with:
           version: "1.10.0"
 
+      - name: Install doxyqml
```

Comment:
Consider using a docker image instead (if it exists).

Reply:
Not that I am aware of. That's why I went with `pip install` as the recommended install path.